.github/scripts/backport/main.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -e
+
+# Setting up Committer identity.
+git config --global user.email "noreply@github.com"
+git config --global user.name "GitHub Actions"
+
+# Get backport label from all the labels of the PR
+backport_label=""
+branch_name=""
+for label in "${LABELS[@]}"; do
+    echo "each label - $label"
+    if [[ $label == "backport"* ]]; then
+        backport_label=$label
+        branch_name=${label#backport }
+        break
+    fi
+done
+
+# Exit if backport label not found
+echo "Label Name: $backport_label"
+if [ -z "$backport_label" ]; then
+    echo "Backport label not found. Exiting."
+    exit 1
+fi
+
+# Checkout the version branch
+git fetch origin "$branch_name"
+git checkout "$branch_name"
+echo "Version branch: $branch_name"
+
+# Checkout new backport branch
+new_branch="backport/$ISSUE_NUMBER"
```

Comment:
We should include the label in the branch name.

Reply:
Sure, as the label will have format like `backport <branch_name>`, then we can name the backport branch like `backport/<branch_name>/<issue_number>`